backend/services/evidence_test.go

```diff
@@ -373,39 +368,13 @@ func validateEvidenceSets(t *testing.T, dtoSet []dtos.Evidence, dbSet []FullEvid
 }
 
 func toPtrTagList(in []dtos.Tag) []*dtos.Tag {
-	rtn := make([]*dtos.Tag, len(in))
-
-	for i := range in {
-		rtn[i] = &in[i]
-	}
-
-	return rtn
+	return helpers.Map(in, helpers.Ptr[dtos.Tag])
```

Comment:
Refactored these two methods because the generics we've created previously do exactly the same thing.